backend/app/views/backoffice/base/_menu.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="relative" data-controller="menu">
+  <button data-action="menu#toggle click@window->menu#hide" class="flex items-center rounded-full font-sans transition-all focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 disabled:opacity-60 disabled:pointer-events-none px-0 py-0 focus-visible:outline-green-dark" role="button" tabindex="0" id="react-aria-30" aria-haspopup="true" aria-expanded="true" aria-controls="react-aria-31" style="user-select: none;">
```

Comment:
Please remove all `react-...` placed ids. Either remove aria altogether or generate the own IDs to connect various html elements.